### PR TITLE
Log when staff access ULM

### PR DESCRIPTION
**Pull request by keirog**
So we can see how many people we need to contact when we turn this app
off 😬

 🐿 v2.8.0

### CODE REVIEW

File: whitelist.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 'use strict';
 
+const logger = require('@financial-times/n-logger').default;
 const mongojs = require('mongojs');
 const co = require('co');
-const ACCESS = 'Access denied..';
+const ACCESS = 'Access denied…';
```
Comment (AbdullahAli) [inline whitelist.js]: Was this auto-formatted by the editor?

File: whitelist.js [hunk 2]
```diff
@@ -27,6 +28,12 @@ module.exports = (connectionString) => {
     co(function* () {
       let username = request.cookies.s3o_username;
       let exists = yield findUser(username);
+
+      logger.info({
+        event: 'USER_ACCESSED_ULM',
+        username
```
Comment (AbdullahAli) [inline whitelist.js]: good shout👍🏼

File: whitelist.js (hunk 1 above)
Comment (keirog) [reply whitelist.js]: No I just couldn’t stand to see `..` 😅

Comment (AbdullahAli) [reply whitelist.js]: 😂😂😭

**Pull request merged**